atsign-foundation/noports
# 758 provide more useful messages in the UI when a session is being created

**- What I did**
Fixes #758 
Fixes #709 
**- How I did it**
-  Used progressStream to provide user with useful messages in the UI
- Added authenticateClientToRvd, authenticateDeviceToRVD and encryptedRvdTraffic to the profile form
- Updated tooltip message use same message as the cli for Sshnp arguments

**- How to verify it**

1. Run the app 
2. navigate to profile form screen, the new form screen should be present
3. try to ssh, the terminal screen should display more useful information to the user.

**- Description for the changelog**

- Updated profile form
- updated messages shows to user during ssh
